ucm2/rockchip_es8316/HiFi.conf

```diff
@@ -0,0 +1,70 @@
+# Reference:
```

Comment:
Move configuration to ucm2/Rockchip tree. Create only symlink from ucm2/conf.d tree for the lookups based on the driver name. The Intel configurations will be moved to Intel subtree soon, too.

Reply:
I updated my alsa environment to 1.2.5.  Then, test the configurations in ucm2/Rockchip.  Have the symlink from ucm2/conf.d as well.
It works now.